### PR TITLE
Add README to dimension reduction folder

### PR DESCRIPTION
**Pull request by cansav09**
## Purpose:
This PR addresses issue #39 which is the lack of README for the dimension reduction module. 

## Pull Request Check List:
* [x] Comments and/or documentation up to date
* [x] Check headline numbering
* [x] Spell check any Rmd file or md file
* [x] Restart R and run all notebooks fresh and save

### CODE REVIEW

File: dimension-reduction/umap_example.Rmd [hunk 1]
```diff
@@ -10,6 +10,12 @@ output:
 *Purpose*: This notebook illustrates how to perform Uniform Manifold Approximation
 and plot the coordinates using ggplot2. 
 
+UMAP is a newer dimension reduction technique that can seemingly yield more
```
Comment (jaclyn-taroni) [inline dimension-reduction/umap_example.Rmd]: Can you tell me more about what you mean by this?

File: dimension-reduction/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Dimension Reduction
+
+In this module, we'll demonstrate how to use dimension reduction techniques with
+gene expression data obtained from [refine.bio](https://www.refine.bio/).
+Dimension reduction is useful for examining overall patterns of your data.
```
Comment (jaclyn-taroni) [inline dimension-reduction/README.md]:
```suggestion
Dimension reduction is useful for examining overall patterns in your data.
```

File: dimension-reduction/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+Reducing your data to two dimensions allows you to more easily plot your samples
+so you can observe global and local patterns in your dataset.
```
Comment (jaclyn-taroni) [inline dimension-reduction/README.md]: What do you mean by this?

File: dimension-reduction/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## Principal Components Analysis (PCA)
+
+This notebook illustrates how to perform PCA and plot the scores using ggplot2.
```
Comment (jaclyn-taroni) [inline dimension-reduction/README.md]: Needs link to rendered HTML notebook https://alexslemonade.github.io/refinebio-examples/dimension-reduction/pca_example.nb.html

File: dimension-reduction/README.md (hunk 3 above)
Comment (cansav09) [reply dimension-reduction/README.md]: You can see how samples spread around overall, and you can also see how samples group (or don't group together) ie what's their neighbors, vs also overall structure.

Comment (jaclyn-taroni) [reply dimension-reduction/README.md]: Can you rewrite this sentence to include that information?

**Pull request merged**